benchmark/logic_rust/concurrent_deser.rs

```diff
@@ -0,0 +1,129 @@
+use chrono::Local;
+use futures::future::join_all;
+use scylla::client::session::Session;
+use scylla::client::session_builder::SessionBuilder;
+use scylla::statement::prepared::PreparedStatement;
+use scylla::value::CqlTimeuuid;
+use std::env;
+use std::net::{IpAddr, Ipv4Addr};
+use std::str::FromStr;
+use std::sync::Arc;
+use uuid::Uuid;
+
+const CONCURRENCY: usize = 2000;
```

Comment:
:question: :thinking: Here I can see `CONCURRENCY` hard-coded. I don't see an equivalent constant in JS benchark code. Where is it?